providers/implementations/kdfs/krb5kdf.c

```diff
@@ -275,7 +275,7 @@ static int fixup_des3_key(unsigned char *key)
  *
  * block = 0
  * for k: 1 -> K
- *   block += s[N(k-1)..(N-1)k] (one's complement addition)
+ *   block += s[N(k-1)..(N-1)k] (ones'-complement addition)
```

Comment:
To hyphenate or not to hyphenate?

You've removed the hyphen in _two's complement_, added it here and left it left in the change in `include/internal/numbers.h`.

I'd suggest removing it here to be consistent.

Reply:
> To hyphenate or not to hyphenate?
> 
> You've removed the hyphen in _two's complement_, added it here and left it left in the change in `include/internal/numbers.h`.
> 
> I'd suggest removing it here to be consistent.

@paulidale according to [review](https://github.com/openssl/openssl/pull/26095#discussion_r1868887372)

Reply:
You should then put one in in `include/internal/numbers.h` where you changed the spelling but didn't hyphenate.

Reply:
No, when the compound is on its own, it doesn't need a hyphen: `ones' complement` on its own is fine. But when used as an adjective, as in `ones-complement system`, it does need to be hyphenated. Yay for English :(